schedule/yast/sle/yast2_cmd.yaml

```diff
@@ -0,0 +1,8 @@
+name:           yast2_cmd
```

Comment:
Could you please follow same convention that Alex did in recently merged #8268 to name and locate your yaml?

Reply:
Done